Dockerfile

```diff
@@ -36,21 +36,28 @@ RUN wget -O /cubit.deb https://f002.backblazeb2.com/file/cubit-downloads/Corefor
 # install cubit
 RUN dpkg -i cubit.deb
 ENV PYTHONPATH=/opt/Coreform-Cubit-2023.11/bin/
+COPY ./rlmcloud.in /opt/Coreform-Cubit-2023.11/bin/licenses/rlmcloud.in
+
+RUN mkdir -p /opt/etc
+RUN cp /root/.bashrc /opt/etc/bashrc
 
 # parastell env
 COPY ./environment.yml /environment.yml
 RUN conda env create -f environment.yml
-RUN echo "source activate parastell_env" >> ~/.bashrc
+RUN echo "conda activate parastell_env" >> /opt/etc/bashrc
 
 WORKDIR /opt
 
 # install pystell_uw
 RUN git clone https://github.com/aaroncbader/pystell_uw.git
 ENV PYTHONPATH=$PYTHONPATH:/opt/pystell_uw
 
-# install parastell
-RUN git clone https://github.com/svalinn/parastell.git
-ENV PYTHONPATH=$PYTHONPATH:/opt/parastell
-
 WORKDIR /
 
+from parastell-deps as parastell
+
+# install parastell
+RUN git clone https://github.com/svalinn/parastell.git && \
+    cd parastell && \
+    git checkout gh-action
```

Comment:
Does the action already do a checkout via `actions/checkout@v4`? I'm still learning GitHub Actions so this is slightly unclear to me

Reply:
I've removed this block from the Dockerfile and everything seems to work. Is that bad practice?

Reply:
No - this is OK! and probably better.  We don't actually need to install parastell in the Dockerfile.  The only reason to keep it may be to have a Dockerfile that can be used by others to make an image for Parastell?

Reply:
And if we did want that, we might want to have a stage that stops short of installing parastell that we use for the CI image we use here.

Reply:
But this Dockerfile is already tweaked so much to work in our particular way that it isn't a good basis for making a personal docker image anyway